Lib/tkinter/__init__.py

```diff
@@ -7,15 +7,15 @@
 LabelFrame and PanedWindow.
 
 Properties of the widgets are specified with keyword arguments.
-Keyword arguments have the same name as the corresponding resource
+Keyword arguments have the same name as the corresponding options
 under Tk.
 
 Widgets are positioned with one of the geometry managers Place, Pack
 or Grid. These managers can be called with methods place, pack, grid
 available in every Widget.
 
-Actions are bound to events by resources (e.g. keyword argument
-command) or with the method bind.
+Actions are bound to events by options (e.g. keyword argument
+command) or with the bind() method.
```

Comment:
'command' needs to be marked as literal string.
```suggestion
`command`) or with the bind() method.
```
Not sure if suggestion is correct, but as is, I initially read this 'keyword-option-command' as 1 compound noun.

Also, line 28, in the example, which is what got me to parse this correctly, needs 2 spaces after commas.

Reply:
What about `the command keyword argument`?

Reply:
OK